core/src/main/java/io/questdb/cutlass/line/CairoLineProtoParserSupport.java

```diff
@@ -93,6 +94,15 @@ public static void putValue(TableWriter.Row row, int columnType, int columnIndex
                 case ColumnType.TIMESTAMP:
                     row.putTimestamp(columnIndex, Numbers.parseLong(value, 0, value.length() - 1));
                     break;
+                case ColumnType.GEOHASH:
+                    if (value.length() == 2) {
```

Comment:
can we perhaps add a comment of what magic number 2 is? I'm assuming quotes but it would be good if we rule out guess work